### PR TITLE
fix: the whitespace on auth screen

### PR DESCRIPTION
**Pull request by akbansa**
## Problem

https://github.com/PostHog/posthog/issues/8768

## Changes

👉 *Stay up-to-date with our [coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*
Screenshot 2022-03-07 at 5 23 40 PM

## How did you test this code?

- I execute the frontend test cases successfully.
- I tested it manually as it was a visual change on one of the UI element

### CODE REVIEW

**Comment by pauldambra** [comment]
Thanks for the contribution, @akbansa!

I've tagged @clarkus for UI review. 

I'll grab a screenshot of the figma you mention not having access to in the issue too

**Comment by pauldambra** [comment]
Here's the design with the spacing highlighted

Screenshot 2022-03-07 at 12 09 41

File: frontend/src/scenes/authentication/Login.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
 .bridge-page.login {
     .helper-links {
         text-align: center;
+        padding-top: 1rem;
```
Comment (pauldambra) [inline frontend/src/scenes/authentication/Login.scss:7]: we're already importing the `vars.scss` in this file so could replace `1rem` with `$default_spacing`

**Review by pauldambra — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/authentication/Login.scss (hunk 1 above)
Comment (pauldambra) [reply frontend/src/scenes/authentication/Login.scss:7]: adding another commit should also fix the linting error on the PR title :)

**Review by pauldambra — APPROVED** [review]

**Comment by pauldambra** [comment]
looks good to me @akbansa 

will wait for @clarkus to come online and review

**Review by clarkus — APPROVED** [review]

**Comment by pauldambra** [comment]
That's merged in. Thanks for the contribution @akbansa 🙌

**Pull request merged**